### DISCUSSION TITLE
[Just an idea] Provide a simple step-by-step tutorial, showcasing one application via wxruby3 that is both simple and useful, in a tutorial-centric manner

### DISCUSSION THREAD

**Discussion by rubyFeedback** [mcorino/wxRuby3#338] [Ideas]
I would like to suggest that eventually a simple step-by-step tutorial is added to wxruby3.

This should feature SIMPLE widgets - one or more labels (text), one or more buttons, 
possibly how to lay these components out (a grid, ideally), also an entry / textview
component for obtaining user input ... that's about it.

The tutorial should be simple, in a step-wise manner. You don't have to be super-verbose
in description, just a few pointers; people can then read up more on their own.

IF such a tutorial already exists, and includes the above points, then please ignore this
issue request. As this is not a high priority thing, and can be done in months or even
years from here on, I will close it. (I'll eventually learn stuff on my own as I research
the wxwidgets documentation, but it would be a lot simpler if wxruby itself could provide
a starter. Right now I don't know how to layout boxes, for instance.)

**Comment by mcorino** [comment]
@rubyFeedback Have you looked at the User Guide and Howto-s on the Wiki? (still a work in progress; contributions welcome).

Of course if you have an idea about a larger, integrated tutorial that's ok. All contributions are welcome.
